src/app/space/app-launcher/services/app-launcher-pipeline.service.ts

```diff
@@ -43,6 +43,7 @@ export class AppLauncherPipelineService implements PipelineService {
       return this.http.get(runtimeEndPoint, option)
         .map(response => response.json() as Pipeline[])
         .map(pipelines => {
+          console.log(pipelines);
```

Comment:
Is this needed? Prefer to be removed if possible.

Reply:
not at all. shame on me. we have to get some bot to catch that.

Reply:
I will fix that after the rebase. :beers: @jiekang

Reply:
Done in b5f581303d48265cc04704f46a5ce5aa88486c03